content/module-reference/utility-modules/shared/tagging.md

```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
 - A tag set as “category” is not exported in `XMP-dc Subject`. However it is exported in `XMP-lr Hierarchical Subject` as this XMP metadata holds the organization of your tags.
 
 - “synonyms” enrich the tag information and are mainly used to assist search engines. For example “juvenile”, “kid” or “youth” can be set as synonyms of “child”. Synonyms can also be used to translate tag names to other languages.
- 
-The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the catagories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
+
+The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the categories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
+
+A quick way to organize the tag structure is the drag and drop. In the _tree_ view mode, you can drag whatever tag (node or leave) and drop it on top of another one (node or leave). The first one and its descendants if any become descendants of the second one. Dragging over a node automatically opens the node.
+To place a tag at the root level, drag it onto the top of the tagging window to make appear a temporary record "drop to root" where you can drop it.
```

Comment:
Can this be shorter?

Such as: To place a tag at the root level, drag it onto the top of the tagging window.

Reply:
The tagging window has no room to drop a tag to root, the root not being represented.
The trick here is to make appear this "drop to root" tag to allow this operation. 
And to make appear this temporary tag, the user needs to drag a tag, move the mouse to the top of the window. Only then the user can release his tag on top of it and move effectively a tag on the root of the tree. Once done the temporary "drop to root" tag disappears.

Reply:
If you think this is enough, let's do it.